setup.py

```diff
@@ -1,5 +1,13 @@
+import os
+
 import setuptools
 
+# switch entry point depending on the debugger during development
+if os.getenv("ENVIRONMENT") == "dev":
```

Comment:
Quick drive-by: this env name is very generic, I can easily see this being set by unrelated software by accident. It should be way more specific to osbuild/setup to avoid any chance that this is triggerd by chance (e.g. OSBUILD_SETUP_INSTALL_DEBUG_ENVIRONMENT or eqully big mouthfulls. Also have you considered just "patching" inside the Containerfile instead of putting it into this generic code? It seem to come back to what @achilleas-k wrote, this is now modifying generic code to cater for a specific workflow that is (at least currently) very specific? Isolating as much as possible inside the dockerfile will probably make this PR easier

Reply:
the variable `OSBUILD_SETUP_INSTALL_DEBUG_ENVIRONMENT` feels very long, but readable, I'll take this suggestion :-D 
I can't think of a "patch" that will be somewhat stable and also I really try to avoid changing files that might be committed later on (although it would be in the container). In the current implementation the generic code is not changed, just a wrapper is added during pip install (protected with this noone_will_ever_catch_this_variable_name 😄👌) . I think that's more straight forward and less fragile

Reply:
…well I did a patch now :-D